MIGRATION.md

```diff
@@ -2,7 +2,77 @@
 
 The guides below are provided to ease the transition of existing applications using the Onfido SDK from one version to another that introduces breaking API changes.
 
-## `5.10.0` -> [next-version]
+## `5.10.0` -> `6.0.0`
+
+### Change in UX flow for Document step
+
+- Document step now has a Issuing Country Selection screen after the Document Type Selection screen. This screen is not displayed for **passport** documents. It is also skipped by default if the Document step is configured with driving licence or national ID card as the only preselected document type. This screen can still be included in the document capture flow by enabling the `showCountrySelection` option in the Document step configuration.
```

Comment:
I found this description a bit confusing. How about

> [...]. This screen is never displayed for **passport** documents and is disabled by default when documents are preselected using the `documentTypes` option. This screen can still be included in the document capture flow of **non-passport** preselected documents by enabling the `showCountrySelection` option in the Document step configuration.

Reply:
That sounds much better, thanks. Was struggling with how to document this. Though I think it's still important to be clear that it is only disabled by default if only 1 document is preselected.